CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
```

Comment:
It should probably contain a "Breaking Change" note that NM 1.22 is now required.

Reply:
Done, thanks!